runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
+
+Verifies an LMS signature. The hash to be verified is taken from
+Caliptra's SHA384 accelerator peripheral.
+The supported parameter set is limited to: sha256/192, w=4, and h=15 (the same used for the Caliptra image)
+
+Command Code: `0x4C4D_5356` ("LMSV")
+
+*Table: `LMS_SIGNATURE_VERIFY` input arguments*
+
+| **Name**              | **Type** | **Description**
+| --------              | -------- | ---------------
+| chksum                | u32      | Checksum over other input arguments, computed by the caller. Little endian.
+| pub\_key\_tree\_type  | u8[4]    | LMS public key algorithm type. Must equal 12.
+| pub\_key\_ots\_type   | u8[4]    | LM-OTS algorithm type. Must equal 7.
```

Comment:
Should we list the enum values under this command heading so they seem less magic? Also if they're spec defined, hopefully folks will be less likely to change them in the code :)

Reply:
In the description? Yeah I had considered that and just didn't end up doing it. I'll make that change.